shoop/core/models/_service_behavior.py

```diff
@@ -168,3 +172,39 @@ def get_unavailability_reasons(self, service, source):
         groups_to_match = set(self.groups.all().values_list("pk", flat=True))
         if not bool(customer_groups & groups_to_match):
             yield ValidationError(_("Service is not available for any of the customers groups."))
+
+
+class RoundingBehavior(Enum):
+    ROUND_UP = 1
+    ROUND_DOWN = 2
+
+    class Labels:
+        ROUND_UP = _("round up")
+        ROUND_DOWN = _("round down")
+
+
+class RoundingBehaviorComponent(ServiceBehaviorComponent):
+    name = _("Rounding")
+    help_text = _("Round total order price to the nearest quant.")
+
+    rounding_quant = models.DecimalField(
+        max_digits=36, decimal_places=9,
+        verbose_name=_("rounding quant"))
```

Comment:
I think `Decimal('0.05')` would be a good default value for the `rounding_quant`.